backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlConfig.java

```diff
@@ -97,6 +98,13 @@ public DataFetcherExceptionResolver dataFetcherExceptionResolver() {
         return Mono.just(singletonList(new GenericGraphqlException(errorMessage, errorPath)));
       }
 
+      if (exception instanceof UnidentifiedFacilityException) {
```

Comment:
TIL about this file that sends a more descriptive error message to the frontend. Only moved the new error that I made for this specific case here and not the more generic `IllegalArgumentException` here to be sent to the frontend (handling it instead in the resolver) so as to not send all instances of the more generic exception to the frontend.